### DISCUSSION TITLE
How to connect Zigbee2mqtt to a ZiGate-Ethernet?

### DISCUSSION THREAD

**Discussion by pce5962** [Koenkk/zigbee2mqtt#16893] [Devices]
Good morning,

I installed my PiZiGate+ in a Raspberry PI2, on which I configured socat to make the whole thing a ZiGate-Ethernet.

Today, in a virtual machine, I want to install Mosquitto MQTT and Zigbee2mqtt, but I don't see how to configure the connection between Zigbee2mqtt and the ZiGate-Ethernet.
In Domoticz, the Zigbee for Domoticz plugin is configured to access the Zigate-Ethernet on tcp://MYIPPI2:9999, where should this parameter be set to establish this connection?

Philippe

**Comment by lux73** [comment]
configuration.yaml

```
serial:
    port: 'tcp://192.168.2.13:20108'
```
https://www.zigbee2mqtt.io/guide/configuration/#configuration

https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html#_3-configure

be aware the config is very case-sensitiv (such as any YAML config) - any false space will prevent z2m to start
